scripts/debian/build.sh

```diff
@@ -9,7 +9,7 @@ SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 # In case of running this script on detached head, script has difficulties in finding out
 # what is the current branch.
 if [[ -n "${BRANCH_NAME}" ]]; then
-  BRANCH_NAME="${BRANCH_NAME}" source "${SCRIPTPATH}"/../export-git-env-vars.sh
+  source "${SCRIPTPATH}"/../export-git-env-vars.sh
```

Comment:
Aren't you introducing a bug here? in `buildkite/scripts/export-git-env-vars.sh`, this is indeed being use.